utils/polus-imagej-util/{{cookiecutter.project_slug}}/plugin.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "{{ cookiecutter.project_name }}",
+  "version": "{{ cookiecutter.version }}",
+  "title": "{{ cookiecutter.project_name }}",
+  "description": "{{ cookiecutter.project_short_description }}",
+  "author": "{{ cookiecutter.author }} ({{ cookiecutter.email }})",
+  "institution": "National Center for Advancing Translational Sciences, National Institutes of Health",
+  "repository": "https://github.com/labshare/polus-plugins",
+  "website": "https://ncats.nih.gov/preclinical/core/informatics",
+  "citation": "",
+  "containerId": "labshare/{{ cookiecutter.project_slug }}:{{ cookiecutter.version }}",
```

Comment:
This should be changed to `polusai/`. All of our plugins are going to be hosted by the `polusai` group rather than `labshare`.